easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2021b.eb

```
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'scanpy'
+version = '1.9.1'
+
+homepage = 'https://scanpy.readthedocs.io/en/stable/'
+description = """Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
+ jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
+ and differential expression testing. The Python-based implementation efficiently deals with
+ datasets of more than one million cells.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('h5py', '3.6.0'),
+    ('networkx', '2.6.3'),
+    ('numba', '0.54.1'),
+    ('PyTables', '3.6.1'),
+    ('statsmodels', '0.13.1'),
+    ('scikit-learn', '1.0.1'),
+    ('Seaborn', '0.11.2'),
+    ('tqdm', '4.62.3'),
+    ('leidenalg', '0.8.8'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('natsort', '8.2.0', {
+        'checksums': ['57f85b72c688b09e053cdac302dd5b5b53df5f73ae20b4874fcbffd8bf783d11'],
+    }),
+    # anndata 0.7.8 needs xlrd<2.0
+    ('xlrd', '2.0.1', {
+        'checksums': ['f72f148f54442c6b056bf931dbc34f986fd0c3b0b6b5a58d013c9aef274d0c88'],
+    }),
+    ('anndata', '0.8.0', {
```

```suggestion
    ('xlrd', '2.0.1', {
        'checksums': ['f72f148f54442c6b056bf931dbc34f986fd0c3b0b6b5a58d013c9aef274d0c88'],
    }),
    ('anndata', '0.8.0', {
```

The comment is irrelevant?